controllers/healthcheck_test.go

```diff
@@ -0,0 +1,31 @@
+package controllers
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gorilla/mux"
+	"github.com/prest/prest/testutils"
+)
+
+func TestHealthStatus(t *testing.T) {
+	var testCases = []struct {
+		description string
+		url         string
+		method      string
+		status      int
+	}{
+		{"Get databases without custom where clause", "/_health", "GET", http.StatusOK},
```

Comment:
1. Aren't we testing the failure?
2. If not, do we need a test cases table for a single test?

Reply:
So, to test the failure I need to mock the database connection and I'm not finding a suitable way to do it. Can you help me with that?

But yes, we **are not** testing for failure

Reply:
I cannot. I haven't dug that further on the abstraction of the `testutils.DoRequest`. What I would do if I were on your shoes:

1. take a look at other tests just in case they are testing HTTP failures somewhere else (maybe you've covered that already, np)
2. as on pREST community
3. wait or call for opinions on other reviewers here : )

Reply:
@avelino @arxdsilva can you guys help us here?